src/core/utils/index.js

```diff
@@ -192,6 +193,13 @@ export function getErrorComponent(status) {
   }
 }
 
+export function removeProtocolFromURL(urlWithProtocol) {
+  invariant(urlWithProtocol, 'url is required');
```

Comment:
`url` should be `urlWithProtocol`

Reply:
I guess... really it's just saying the argument should be truthy; as it's not a named argument I thought "url" was more helpful. But I guess `urlWithProtocol` will be easier to grep.